README.md

```diff
@@ -1,140 +1,27 @@
 
-```
-make run
-```
+### Contributor License Agreements
 
-This will bring up a local development server.
-
-```
-make serve-front-end
-```
-
-This will serve the frontend.
-
-## Environment Variables
-
-We check for the presence of some environment variables to control various aspects of building and testing:
-
-| Environment variable     | Description                                       |
-| ------------------------ | ------------------------------------------------- |
-| RUN_IMAGE_SNAPSHOT_TESTS | Enable/disable image snapshots during testing.    |
-| GITHUB_WORKFLOW          | Enable/disable certain tests in a CI environment. |
-
-## Running the project from Xcode
-
-Alternatively, you can open the `Package.swift` file in Xcode and run the server from there. However, it's important to set a custom working directory before running. To do this, navigate to the **Product** | **Scheme** | **Edit Scheme...** menu or press ⌘+<. Select the **Run** scheme action and select the **Options** tab. Finally, check the **Working Directory** checkbox and enter the directory where you have this source code checked out.
-
-## Resetting the database
-
-You can reset the database to a clean slate by tearing down the containers by running:
-
-```
-make db-reset
-```
-
-This command will _destroy_ both the development, and test docker containers. Recreate them, and finally migrate the development database.
-
-## Running an end-to-end test
-
-Once you're all set up, run:
-
-```
-make test-e2e
-```
-
-This will kick off a local test run of the server update process (reconciliation, ingestion, and analysis) processing just _one_ package. This is a good way to verify everything is working.
-
-## Running ingestion locally
-
-The `ingest-loop.sh` script can serve as a simple way to run a full ingestion cycle:
-
-```
-make reset        # clear dev db
-make reconcile    # import package list
-./scripts/ingest-loop.sh  # ingest metadata for 100 packages, pause for 10 sec, repeat
-```
-
-If you want to run ingestion for anything other than a cursory test, you'll need authenticated API calls. To do this, set a `GITHUB_TOKEN` environment variable to a [generated personal token](https://github.com/settings/tokens) which has the `public_repo` and `repo:status` scopes.
-
-## Running analysis locally
-
-You can run the analysis step locally by running
-
-```
-make analyze
-```
-
-This will run the analysis stage for one package (`--limit 1`).
-
-NB: The analysis step will check out repositories to your local file system, by default into a directory `SPI-checkouts` in your project folder. You can change this location by setting the environment variable `CHECKOUTS_DIR` to another path.
-
-## API poking
```

Comment:
I've deliberately removed this section. It didn't feel particularly useful, but we can put it back if you feel strongly about it.

Reply:
Nah, that's a remnant from when we had an API :)